solpolpy/polarizers.py

```diff
@@ -28,7 +35,7 @@ def npol_to_mzp(input_cube):
     mzp_ang = [-60, 0, 60]
     Bmzp = {}
     for ang in mzp_ang: Bmzp[ang * u.degree] = (1 / 3) * np.sum(
-        [ith_polarizer_brightness * (1 + 2 * np.cos(2 * (ang * u.degree * conv_fact - ith_angle)))
+        [ith_polarizer_brightness * (1 + 2 * np.cos(2 * (ang * u.degree * conv_fact - (ith_angle-offset_angle* u.degree * conv_fact))))
```

Comment:
You don't need this `conv_fact`. See [this documentation](https://docs.astropy.org/en/stable/coordinates/angles.html#id2) for a cleaner way to handle radians and degree conversion. Astropy.units is smart so use it.